typescript/src/bitcoin/network.ts

```diff
@@ -0,0 +1,66 @@
+import { TransactionHash } from "../bitcoin"
```

Comment:
Can we keep a flat file structure for now? I propose to name this file `bitcoin-network.ts` and place it in `src` dir directly (same for the test file). We will probably refactor the file structure anyway soon so it would be good to start from a consistent approach.

Reply:
3f1b7b3f2cfd43a894970a5f167c449ecbb9f228